### DISCUSSION TITLE
How to change repo for updates

### DISCUSSION THREAD

**Discussion by vale-max** [rern/rAudio#433] [General]
Hello @rern 
as I have made some changes here and there to better suit my setup (DAB radio, wireless and bluetooth on RPi1) I would like to have rAudio pick the updates from my repo, so I can merge your updates with main and avoid having to do it manually after the update.

What would be the files to change to  change the source repo?

**Comment by rern** [comment]
@vale-max,

File: `/srv/http/data/addons/addons-list.json`
- Change `https://github.com/rern/rAudio-1` to your repo.
- On your GitHub repo, create a release with the `version` number as `Tag version`

Note: On every `Settings` > `Addons`, the file will be overwritten.
